pkg/azureupdate/error.go

```diff
@@ -12,3 +12,12 @@ var invalidConfigError = &microerror.Error{
 func IsInvalidConfig(err error) bool {
 	return microerror.Cause(err) == invalidConfigError
 }
+
+var cantChangeMasterCIDRError = &microerror.Error{
```

Comment:
Pure nitpick, matter of taste etc.:

"Accented" (i.e. `can't`, `won't`) language feels a bit wrong in code so could this be named a bit differently:
`masterCIDRChangeError` w/ message "disallowed":

```go
microerror.Maskf(masterCIDRChangeError, "disallowed")
```

...or more verbosely as below:
```go
microerror.Maskf(masterCIDRChangeError, "Spec.Azure.VirtualNetwork.MasterSubnetCIDR change disallowed")
```

Reply:
That's a good suggestion, thanks.
Renamed.